### PR TITLE
style: building taskcard and itemtile ui components

### PR DESCRIPTION
**Pull request by DavidZZGi**
## Type of Change

- [ ] ✨ New feature (non-breaking change which adds functionality)
- [ ] 🛠️ Bug fix (non-breaking change which fixes an issue)
- [ ] ❌ Breaking change (fix or feature that would cause existing functionality to change)
- [ ] 🧹 Code refactor
- [ ] ✅ Build configuration change
- [ ] 📝 Documentation
- [ ] 🗑️ Chore

### CODE REVIEW

File: packages/app_ui/lib/src/widgets/priority_widget.dart [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+
+      case 5:
+        priorityIcon = const Icon(
+          Icons.keyboard_double_arrow_down,
+          color: Colors.blue,
+        );
+        priorityText = 'Lowest';
+    }
+    final avatar = CircleAvatar(
+      backgroundColor: Theme.of(context).colorScheme.primary.withOpacity(0.3),
```
Comment (yarn-rp) [inline packages/app_ui/lib/src/widgets/priority_widget.dart:69]: let'c create a variable with the `final theme = Theme.of(context);` at the top en reuse that theme on the widget

**Review by yarn-rp — CHANGES_REQUESTED** [review]
Nice additions, but it's very hard for me to see if these widgets are ok or not. This is exactly why the storybook is nice, but since we don't have it, would you mind to upload screenshoots of the widgets with the name, so it's easier to see if they are right or not.

Also when doing Paddings and Margins, lets try to use `AppSpacing` class to define the sizes instead of using raw values. Related to this, kinda curious what are you doing to set the margin and the padding? Is this generated from Figma?

File: packages/app_ui/lib/src/widgets/item_tile.dart [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+  });
+  final String title;
+  final String subtitle;
+  final int priority;
+  final String btnText;
+  final Color backgroundColor;
+
+  @override
+  Widget build(BuildContext context) {
+    // ignore: use_colored_box
```
Comment (yarn-rp) [inline packages/app_ui/lib/src/widgets/item_tile.dart:21]: Don't ignore linter rules if not necessary

**Comment by DavidZZGi** [comment]
Screenshot TaskCard
Screenshot 2023-11-02 at 8 40 06 AM

**Pull request closed without merging**